{
  try {
    template.sendBody("direct:start","Hello World");
    fail("Should have thrown exception");
  }
 catch (  CamelExecutionException e) {
    IOException cause=assertIsInstanceOf(IOException.class,e.getCause());
    assertNotNull(cause);
    assertEquals("First failure message is: Damn",cause.getMessage());
  }
}
